client/components/jetpack/threat-dialog/index.tsx

```diff
@@ -90,6 +143,74 @@ const ThreatDialog: React.FC< Props > = ( {
 			<h3 className="threat-dialog__threat-title">
 				<ThreatFixHeader threat={ threat } action={ action } />
 			</h3>
+			{ isExtensionDeleteFixer && (
```

Comment:
I considered extracting this whole thing into its own component, but it is a bit intertwined because it does have to change the buttons of this component as well, so I didn't do that so far 🤷🏻‍♂️